app/models/story.rb

```diff
@@ -14,10 +14,9 @@ def self.paranoia_scope
     )
   end
 
-  belongs_to :account, -> { with_deleted }, touch: true
-  belongs_to :creator, -> { with_deleted }, class_name: 'User', foreign_key: 'creator_id',
-                                            touch: true
-  belongs_to :series, touch: true
+  belongs_to :account, -> { with_deleted }
```

Comment:
When you get an account under the `/authorization` endpoint, aren't stories embedded in it?